keras_nlp/models/__init__.py

```diff
@@ -12,14 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from keras_nlp.models.bert import BertBase
-from keras_nlp.models.bert import BertCustom
-from keras_nlp.models.bert import BertLarge
-from keras_nlp.models.bert import BertMedium
-from keras_nlp.models.bert import BertPreprocessor
-from keras_nlp.models.bert import BertSmall
-from keras_nlp.models.bert import BertTiny
-from keras_nlp.models.bert_tasks import BertClassifier
+from keras_nlp.models.bert.bert_models import BertBase
```

Comment:
Why change these symbols? I thought we are just reshuffling files behind the scenes